### DISCUSSION TITLE
Perhaps fishing in Cindersap Forest on Spring 3 is not best

### DISCUSSION THREAD

**Discussion by nathan-alden-sr** [nathan-alden-sr/stardew-valley#34] [Ideas]
_Note: See the comments for the results of an experiment. It turns out my speculative numbers were off, but the hypothesis still seems to be true: it's better to fish at The Mountain._

I recently started a new run, and today I literally spent _all day_ running Spring 3 over and over and over again. Actually, add one more _over_. I began to wonder, especially for those doing no-reset runs, is it actually better to fish at The Mountain?

Unfortunately, the wiki does not list enough details on the spawn rates for Catfish to compare to Largemouth Bass. Largemouth bass stop spawning at 7:00 pm, while Catfish stop spawning at 12:00 am.

My feeling is that Catfish and Largemouth Bass seem to spawn at similar rates. Fishing all day in Cindersap Forest seems to result in anywhere from 10-15 Catfish hits while a similar, maybe slightly fewer, number of Largemouth Bass can be expected at The Mountain.

Let's assume 15 fish in each location and that we have the Fisher profession before selling fish.

In most of my runs, despite fishing reliably in Fishing Zone 5, Catfish seem to be half-and-half silver and gold quality. It's virtually  impossible without Godlike RNG to perfect-catch Catfish, so that explains the poor quality. Also, some Catfish patterns are virtually impossible to catch, so let's assume 33% of hits end up getting away.

| Fish | Quality | Catches | XP/ea. | Total XP | Sell G/ea. | Total Sell G |
| - | - | - | - | - | - | - |
| Catfish | Miss | 5 |
| Catfish | Silver | 5 | 31 | 155 | 312 | 1,560 |
| Catfish | Gold | 5 | 34 | 170 | 375 | 1,875 |
| **Grand Totals** | | | | **325** | | **3,435** |

In most of my trips to The Mountain, I can reliably catch gold- or iridium-quality Largemouth Bass and never fail to catch one like I do with Catfish. However, it seems to be tough to reliably perfect-catch Largemouth, especially as early as Spring 3, so let's assume a 33% perfect-catch rate. (not relevant to Spring 3)

| Fish | Quality | Catches | XP/ea. | Total XP | Sell G/ea. | Total Sell G |
| - | - | - | - | - | - | - |
| Largemouth Bass | Gold | 10 | 25 | 250 | 187 | 1,870 |
| Largemouth Bass | Iridium | 5 | 31 | 155 | 250 | 1,250 |
| **Grand Totals** | | | | **405** | | **3,120** |

I know there's some speculation in my numbers, but I think these estimations are pretty accurate. I've certainly caught _fewer_ Catfish than 10 on Spring 3--far fewer--and more on very rare occasions. Considering that Chub spawn at both locations and Bullhead are available at The Mountain, it may be worth experimenting going to The Mountain, instead. I am thinking that Bullhead + Carp cancels out Shad + Bream, as well. And, if I may, some candor: catching Catfish at Fishing level 4 _sucks_--it's tilting and very frustrating how varied the patterns can be. I also tend to miss fishing chests on Catfish catches because it's simply too risky giving up green bar progress for a chest.

This is a screenshot of my Year 1, Spring 3 video of my longest playthrough (made it to Year 2), so these results are atypical on the _good_ side:

![image](https://user-images.githubusercontent.com/1189759/118385015-454ae900-b5d9-11eb-8265-a065de4bcf8c.png)

You can see I caught only 11 Catfish, although the distribution was skewed toward gold quality. I remember this day, too. Nearly all the Catfish attempts were actually relatively easy as their patterns were sane. Perhaps daily Luck is involved somehow?

I'm going to try Spring 3 again at The Mountain and report back my findings.

Thoughts @waynn?

**Comment by nathan-alden-sr** [comment]
I performed an experiment just now, going to The Mountain immediately after buying the Copper Ore from Clint. Here are the results compared to the Spring 3 Cindersap Forest day from my longest run:

Starting Fishing XP: 1536

| Area | Total G | Total XP | XP Gained |
| - | - | - | - |
| Cindersap Forest | 5,861 | 3,306 | 1,770 |
| The Mountain | 6,046 | 3,613 | 2,077 |

It turns out my hypothetical Largemouth Bass totals in the OP were very off, but it also turned out not to matter. Remember that both of these attempts were based on the _same save file_, so therefore the same daily Luck.

**Cindersap Forest**

| Fish | Quality | Qty |
| - | - | - |
| Bream | Iridium | 1 |
| Bream | Gold | 5 |
| Catfish | Gold | 4 |
| Catfish | Silver | 4 |
| Chub | Iridium | 5 |
| Chub | Gold | 7 |
| Shad | Iridium | 5 |
| Shad | Gold | 5 |
| **Grand Totals** | | **36** |

**The Mountain**

| Fish | Quality | Qty |
| - | - | - |
| Bullhead | Iridium | 7 |
| Bullhead | Gold | 5 |
| Carp | Iridium | 8 |
| Carp | Gold | 8 |
| Carp | Silver | 2 |
| Catfish | Gold | 1 |
| Chub | Iridium | 6 |
| Chub | Gold | 10 |
| Chub | Silver | 1 |
| Largemouth Bass | Gold | 2 |
| Largemouth Bass | Silver | 3 |
| **Grand Totals** | | **53** |

I also totaled up the results from my longest playthrough's Spring 3. That Spring 3 was significantly better than the one above, yielding more than 7,300g. Why am I not surprised to discover that the Luck for that day was "very happy," whereas the luck for last night's attempts was "mildly perturbed?" Although, if [this post's](https://www.reddit.com/r/StardewValley/comments/4c2gx4/a_comprehensive_guide_to_luck) conclusions are correct then bite rate, fish rarity, and fish quality are _not_ affected by luck.

**Reply by waynn** [reply]
Sorry for the delay, wanted to dig through my files to see if I had any useful data to contribute!

That does seem like a particularly bad Catfish day though. In my previous Spring 3 (and I've only run it 4 times), I've had anywhere from 8 Catfish to 19. So the variance does seem to be a bit higher, but 8 was on the low side. There's definitely luck on the pattern too (not in game luck, real life luck).

You might have to run it a few more times (if you feel like it) to see what the EV is on both sides.
